### DISCUSSION TITLE
Etsy API application rejected

### DISCUSSION THREAD

**Discussion by alaritin213** [etsy/open-api#903] [Q&A] [answered]
Hi everyone,
 I'm trying to get an etsy api key to use for a small script for only my shop. Etsy rejected my application because 
"Generated API key at the request of 3rd party app"
Which I did not do! There is no third party. I am confused on what to do, has anyone else run into this problem/resolved it?

I am not using plugins of any kind, just trying to query my own shop in python directly. The reason shown is incorrect, so I am not sure what to do!

Even when i make public requests:
`requests.get(
	'https://api.etsy.com/v3/application/openapi-ping', 
	headers={'x-api-key': keystring}
)`

i get "invalid API key"

**Comment by Gareth064** [comment, marked as answer]
Email the Etsy Dev team.

developer@etsy.com

**Reply by alaritin213** [reply]
thanks!
